pulsar-client/src/main/java/org/apache/pulsar/client/impl/ConsumerBuilderImpl.java

```diff
@@ -327,6 +327,13 @@ public ConsumerBuilder<T> acknowledgmentGroupTime(long delay, TimeUnit unit) {
         return this;
     }
 
+    @Override
+    public ConsumerBuilder<T> maxAcknowledgmentGroupSize(int messageNum) {
+        checkArgument(messageNum > 0, "acknowledgementsGroupSize needs to be > 0");
+        conf.setAcknowledgementsGroupSize(messageNum);
+        return this;
+    }
```

Comment:
We also need to add a test for the invalid configuration.

Reply:
OK

Reply:
Fixed.
PTAL,thanks! @RobertIndie